src/Content/Text/HTML.php

```diff
@@ -336,8 +336,6 @@ function ($matches) use (&$codeblocks) {
 
 		$message = html_entity_decode($message, ENT_QUOTES, 'UTF-8');
 
-		$message = str_replace(["<"], ["&lt;"], $message);
```

Comment:
can you add two files here:
https://github.com/friendica/friendica/tree/develop/tests/datasets/content/text/html

bug-8075.html **with** "<" text
bug-8075.txt with the encoded, expected content?

thx :)

Reply:
Seems like I can't.

Reply:
This is quite interesting because you did it the right way .. the "<>" text should **not** be removed, but the tests says it will be removed see the result here: https://travis-ci.org/friendica/friendica/jobs/633997421#L421-L426

So are you sure "<>" won't get escaped anymore? @MrPetovan am I missing something for the test here?
https://github.com/friendica/friendica/blob/5cc2dc7ca37f5037bbba7d2fa8cb408d0b37c250/tests/src/Content/Text/HTMLTest.php#L73-L79

Reply:
@annando my fault, I took the wrong testcase!

Under
https://github.com/friendica/friendica/blob/5cc2dc7ca37f5037bbba7d2fa8cb408d0b37c250/tests/src/Content/Text/HTMLTest.php#L57-L60
add these new array content:

```php
			'bug-8075-html-tags => [
				'expectedBBCode' => '<big rant here> I don't understand tests',
				'html' => '<big rant here> I don't understand tests',
			],
```

This will get automatically passed to this test:
https://github.com/friendica/friendica/blob/5cc2dc7ca37f5037bbba7d2fa8cb408d0b37c250/tests/src/Content/Text/HTMLTest.php#L73-L78
which checks the expected result :-)

Reply:
Ok forget it ^^ .. I think I'm currently too stupid to guide you well enough..